src/plugins/mam/Plugin.ts

```diff
@@ -174,6 +174,7 @@ export default class MessageArchiveManagementPlugin extends AbstractPlugin {
    private onMamMessage = (stanza: string): boolean => {
       let stanzaElement = $(stanza);
       let resultElement = stanzaElement.find(`result[xmlns^="urn:xmpp:mam:"]`);
+      let id = resultElement.attr('id');
```

Comment:
There are many ids, so they should be named properly.

Reply:
renamed